docs/examples/kops-tests-private-net-bastion-host.md

```diff
@@ -0,0 +1,379 @@
+```
+
+Because we really want to use a ssh-agent, start it first (this will :
+
+```bash
+eval `ssh-agent -s`
+```
+
+And add your key to the agent with "ssh-add":
+
+```bash
+ssh-add ~/.ssh/id_rsa
+
+Identity added: /home/kops/.ssh/id_rsa (/home/kops/.ssh/id_rsa)
+```
+
+Then, ssh to your bastion ELB FQDN
+
+```bash
+ssh -A admin@bastion-privatekopscluste-bgl0hp-1327959377.us-east-1.elb.amazonaws.com
+```
+
+Or if you want to automate it:
+
+```bash
+ssh -A admin@`aws elb --output=table describe-load-balancers|grep DNSName.\*bastion|awk '{print $4}'`
+```
+
+And from the bastion, you can ssh to your masters or workers:
+
+```bash
+admin@ip-172-20-2-64:~$ ssh admin@ip-172-20-53-10.ec2.internal
+
+The authenticity of host 'ip-172-20-53-10.ec2.internal (172.20.53.10)' can't be established.
+ECDSA key fingerprint is d1:30:c6:5e:77:ff:cd:d2:7d:1f:f9:12:e3:b0:28:e4.
+Are you sure you want to continue connecting (yes/no)? yes
+Warning: Permanently added 'ip-172-20-53-10.ec2.internal,172.20.53.10' (ECDSA) to the list of known hosts.
+
+The programs included with the Debian GNU/Linux system are free software;
+the exact distribution terms for each program are described in the
+individual files in /usr/share/doc/*/copyright.
+
+Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
+permitted by applicable law.
+
+admin@ip-172-20-53-10:~$
+```
+
+**NOTE:** Remember that you can obtain the local DNS names from your "kops validate cluster" command, or, with the "kubectl get nodes" command. We recommend the first (kops validate cluster) because it will tell you who are the masters and who the worker nodes:
+
+
+```bash
+kops validate cluster
+Using cluster from kubectl context: privatekopscluster.k8s.local
+
+Validating cluster privatekopscluster.k8s.local
+
+INSTANCE GROUPS
+NAME                    ROLE    MACHINETYPE     MIN     MAX     SUBNETS
+bastions                Bastion t2.micro        1       1       utility-us-east-1a
+master-us-east-1a       Master  t2.micro        1       1       us-east-1a
+master-us-east-1b       Master  t2.micro        1       1       us-east-1b
+master-us-east-1c       Master  t2.micro        1       1       us-east-1c
+nodes                   Node    t2.micro        2       2       us-east-1a,us-east-1b,us-east-1c
+
+NODE STATUS
+NAME                            ROLE    READY
+ip-172-20-111-44.ec2.internal   master  True
+ip-172-20-44-102.ec2.internal   node    True
+ip-172-20-53-10.ec2.internal    master  True
+ip-172-20-64-151.ec2.internal   node    True
+ip-172-20-74-55.ec2.internal    master  True
+
+Your cluster privatekopscluster.k8s.local is ready
+```
+
+## MAKING THE BASTION LAYER "HIGLY AVAILABLE".
+
+If for any reason "godzilla" decides to destroy the amazon AZ that contains our bastion, we'll basically be unable to enter to our instances. Let's add some H.A. to our bastion layer and force amazon to deploy additional bastion instances on other availability zones.
```

Comment:
Same note about Godzilla, he may be copyrighted, and thus we need a monster.

Reply:
jajaja.. yup you're right he may be copyrighted. I'll take all recommendations and requested changes into account. Putting a common "preflight" section and using chapters is a good idea. I'm going to work on the modifications during the next days, then send a new PR with all changes.